app/assets/javascripts/simulations.js

```diff
@@ -14,5 +14,18 @@ $(document).ready(function() {
       selectElement.id = selectElement.id.replace(/\d+/g, id);
     });
     tableBody.appendChild(rowCopy);
+    setRowsDeletable(true);
   });
+  setRowsDeletable(false);
 });
+
+setRowsDeletable = function(deletable) {
+  $('.simulation-delete-row').prop('disabled', !deletable); // ! because the boolean is for disabled, not enabled.
+};
+
+deleteRow = function(event) { // this gets called in the HTML by the onclick attribute
+  var row = event.target.parentElement.parentElement;
+  if(row.parentElement.children.length <= 2)
```

Comment:
Okay I'll change that too.  I think you can do single statement blocks in JS but let's use `{} ` by convention from now on